server/utils/citations.js

```diff
@@ -1,32 +1,40 @@
 /* eslint-disable import/prefer-default-export */
 import Cite from 'citation-js';
+import { getPubPublishedDate } from 'shared/utils/pubDates';
+import { pubUrl } from 'shared/utils/canonicalUrls';
 
-export const generateCitationHTML = (pubData, communityData) => {
-	/* TODO: This is a v6 debug line only! */
-	return {};
+const getDatePartsObject = (date) => ({
+	'date-parts': [date.getFullYear(), date.getMonth() + 1, date.getDate()],
+});
+
+const collectionKindToCitationJSPart = (kind) =>
```

Comment:
A lot of the green here is code that I copied over from `generateCitationHtml.js` to handle collections — looks like it didn't get refactored into this file.